celery/utils/time.py

```diff
@@ -383,3 +385,20 @@ def utcoffset(time=_time, localtime=_time.localtime):
 def adjust_timestamp(ts, offset, here=utcoffset):
     """Adjust timestamp based on provided utcoffset."""
     return ts - (offset - here()) * 3600
+
+
+def get_exponential_backoff_interval(
+    factor,
+    retries,
+    maximum,
+    full_jitter=False
```

Comment:
Looks like you removed the feature I described in the documentation, where you can pass `retry_jitter` an integer and it will treat it as a maximum deviation from the calculated countdown value. Is that intentional? I imagine that some applications will only want a small amount of jitter applied to their calculated value, which is why I included that feature.